### DISCUSSION TITLE
Using Next.js with chakra ui.  Home.module.css does not work.

### DISCUSSION THREAD

**Discussion by dogukanolgun** [chakra-ui/chakra-ui#3590] [Q&A]
Hi. I cant use styles inside CSS modules. Am I missing a setup?
I need CSS modules because of the transform: scale(0.5) CSS property which I couldn't find anywhere

**Comment by malashevskyi** [comment]
Hi. You can use global styles:
```javascript
  const theme = extendTheme({
    styles: {
      global: {
        ".your-class": {
          transform: "scale(0.5)",
          transformOrigin: "0 0"
        }
      }
    }
  });

  return (
    <ChakraProvider theme={theme}>
      <Box
        className="your-class"
      >
        Pam pam pam pam!
      </Box>
    </ChakraProvider>
  );
```

Or you can use sx attribute on an element you want to style:
```javascript
<Box
  sx={{
    transform: "scale(0.5)",
    transformOrigin: "0 0"
  }}
  >
   Pam pam pam pam!
</Box>
```

**Reply by malashevskyi** [reply]
I gave an example with sx for properties that you cannot find [here](https://chakra-ui.com/docs/features/style-props)

Of course you can use transform as attributes:
```javascript
<Box transform="scale(4)" transformOrigin="0 0">
   Pam pam pam pam!
</Box>
```

**Reply by malashevskyi** [reply]
One more example, if you want to style multiple inner elements only for a single component [codesandbox](https://codesandbox.io/s/generate-squares-0i1b7?file=/src/App.js)
